### ISSUE TITLE
Docker build still failing

### ISSUE THREAD

**Issue by SchrodingersGat** [inventree/InvenTree#4047]
![image](https://user-images.githubusercontent.com/10080325/206944963-198ad028-3786-4784-89cc-b0ddbcc4aa80.png)

Recent changes have not fixed the docker build issues. Need some more attention on this.

**Comment by matmair** [comment]
@SchrodingersGat seems to work now

**Comment by SchrodingersGat** [comment]
Yeah, adding a step to update setuptools within the docker container got it to work. Idk why, but at least now it works.
